src/main/java/com/github/tommyettinger/digital/Hasher.java

```diff
@@ -314,7 +317,7 @@ public static int randomize3Bounded(long state, int bound) {
      * @return a pseudo-random float between 0f (inclusive) and 1f (exclusive), determined by {@code state}
      */
     public static float randomize1Float(long state) {
-        return ((((state = (((state * 0x632BE59BD9B4E019L) ^ 0x9E3779B97F4A7C15L) * 0xC6BC279692B5CC83L)) ^ state >>> 27) * 0xAEF17502108EF2D9L) >>> 40) * 0x1p-24f;
+        return ((((state = (((state * 0x632BE59BD9B4E019L) ^ 0x9E3779B97F4A7C15L) * 0xC6BC279692B5CC83L)) ^ state >>> 27) * 0xAEF17502108EF2D9L) >>> 40) * EPSILON;
```

Comment:
It's interesting that the multiplier used to produce the most valid zero-to-one floats with uniform spacing is also the machine epsilon; I am guessing that this isn't a coincidence.